src/components/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Drawer as AntdDrawer } from 'antd'
+import { ReactElement } from 'react'
+
+import { DrawerProps } from './Drawer.props'
+import { Footer } from './Drawer.Footer'
+import { Title } from './Drawer.Title'
+
+const styles = {
+  footer: { padding: '24px' },
```

Comment:
To prevent this hardcoded value I could use a css module that defines the class, however I'd then need to pass the styles prop to the antd drawer in order to reset the padding to 0.

The implementation would look like the following:


```css
.drawer {
  footer {
    padding: var(--spacing-padding-xl, 24px)
  }
}
```

```ts
import styles from './Drawer.module.css'

const { drawer } = styles

const antdDrawerStyles = { footer: { padding: 0 } }

....

   <AntdDrawer
      className={drawer}
      ....
      styles={antdDrawerStyles}
....
```